IPython/core/prefilter.py

```diff
@@ -510,6 +510,12 @@ def check(self, line_info):
         if not oinfo['found']:
             return None
 
+        ignored_funs = ['b', 'f', 'r', 'u', 'br', 'rb', 'fr', 'rf']
+        ifun = line_info.ifun
+        line = line_info.line
+        if ifun.lower() in ignored_funs and (line.startswith(ifun + "'") or line.startswith(ifun + '"')):
```

Comment:
Fun fact: startswith and endswith can both take a tuple of strings, so `s.startswith((p1, p2))` is equivalent to `s.startswith(p1) or s.startswith(p2)`. That's a minor point, though, so I won't hold this PR up for it.